packages/block-editor/src/components/block-lock/style.scss

```diff
@@ -52,10 +52,17 @@
 .block-editor-block-lock-toolbar {
 	.components-button.has-icon {
 		min-width: $button-size-small + $grid-unit-15 !important;
-		padding-left: 0 !important;
+	}
+}
 
-		&:focus::before {
-			right: $grid-unit-10 !important;
-		}
```

Comment:
This is a hack for a narrow focus style, but it affected the text-only label.  The default focus style looks good as well  :)

![CleanShot 2023-05-24 at 18 35 37](https://github.com/WordPress/gutenberg/assets/240569/e62803a9-16e0-4e4a-9dc1-b71dd8c5aa27)